<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Helicopter_Faults</class>
 <widget class="QWidget" name="Helicopter_Faults">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Helicopter_Faults</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>10</y>
     <width>291</width>
     <height>591</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="0">
     <widget class="QCheckBox" name="Fault_Vibration_Max_right">
      <property name="text">
       <string>отказ предельная вибрация правого двигателя</string>
      </property>
     </widget>
    </item>
    <item row="19" column="0">
     <widget class="QCheckBox" name="Fault_Curs">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>отказ Курсовой системы</string>
      </property>
     </widget>
    </item>
    <item row="18" column="0">
     <widget class="QCheckBox" name="Fault_AviaHorizont">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>отказ левого авиагоризонта</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QCheckBox" name="Fault_Autopilot">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>отказ автопилота</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="Fault_Vibration_right">
      <property name="text">
       <string>отказ повыышенная вибрация правого двигателя</string>
      </property>
     </widget>
    </item>
    <item row="22" column="0">
     <widget class="QCheckBox" name="Fault_ARK">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>отказ АРК</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="Fault_Engine2">
      <property name="text">
       <string>отказ двигателя 2</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QCheckBox" name="Fault_Vibration_Max_left">
      <property name="text">
       <string>отказ предельная вибрация левого двигателя</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QCheckBox" name="Fault_HidroMain">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>отказ основной гидросистемы</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QCheckBox" name="Fault_VU2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>отказ выпрямителя 2</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="Fault_RV_gear">
      <property name="text">
       <string>отказ (разрушение) привода РВ</string>
      </property>
     </widget>
    </item>
    <item row="24" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="16" column="0">
     <widget class="QCheckBox" name="Fault_Fire">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Пожар</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QCheckBox" name="Fault_HidroDiafrag">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>отказдиафрагмы гидротрасформатора</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QCheckBox" name="Fault_HidroSecond">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>отказ резервной гидросистемы</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QCheckBox" name="Fault_Gen1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>отказ генератора 1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="Fault_RV_contrl">
      <property name="text">
       <string>отказ управл. ОШ РВ</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QCheckBox" name="Fault_VU1">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>отказ выпрямителя 1</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QCheckBox" name="Fault_Gen2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>отказ генератора 2</string>
      </property>
     </widget>
    </item>
    <item row="21" column="0">
     <widget class="QCheckBox" name="Fault_Speed">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>отказ указателя скорости</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="Fault_Vibration_left">
      <property name="text">
       <string>отказ повыышенная вибрация левого двигателя</string>
      </property>
     </widget>
    </item>
    <item row="23" column="0">
     <widget class="QCheckBox" name="Fault_BKK18">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>отказ БКК-18</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="Fault_Engine1">
      <property name="text">
       <string>отказ двигателя 1</string>
      </property>
     </widget>
    </item>
    <item row="20" column="0">
     <widget class="QCheckBox" name="Fault_Barometer">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>отказ Барометрическго высотомера</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="Fault_Gear">
      <property name="text">
       <string>отказ трансмиссии</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Ok">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Применить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cancel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отменить</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
